support-logs/readme.md

```diff
@@ -36,3 +38,12 @@ PUT _ingest/pipeline/extractJsonPipeline
     ]
 }
 ```
+
+## Setup
```

Comment:
Maybe add a link to information about SSM in here?

Reply:
good idea, will do